src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -189,7 +189,7 @@ const DetailsWidget = ({
           <div className={styles.label}>
             <FormattedMessage {...MSG.domainDescription} />
           </div>
-          <div className={styles.value}>
+          <div className={`${styles.value} ${styles.descriptionValue}`}>
```

Comment:
I think it's better to extend the `descriptionValue` class with `value` class and then you can apply only `descriptionValue` class to the div

Reply:
I like that! Will do